<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="progress"/>
     </item>
     <item>
      <widget class="QPushButton" name="startStop">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DropWidget" name="stimulus" native="true"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sweep from:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QuantityWidgetController" name="stimulusMinimum"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>to</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QuantityWidgetController" name="stimulusMaximum"/>
       </item>
      </layout>
     </item>
     <item row="0" column="2" rowspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>418</width>
         <height>38</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Number of steps:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="IntegerWidgetController" name="stimulusSteps"/>
       </item>
       <item>
        <widget class="QCheckBox" name="logarithmic">
         <property name="text">
          <string>Logarithmic</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DropWidget" name="measurement" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QuantityWidgetController</class>
   <extends>QDoubleSpinBox</extends>
   <header>gui.quantitywidget_controller.h</header>
  </customwidget>
  <customwidget>
   <class>DropWidget</class>
   <extends>QWidget</extends>
   <header>gui.dropwidget_controller.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IntegerWidgetController</class>
   <extends>QSpinBox</extends>
   <header>gui.integerwidget_controller.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
